patroni/dcs/__init__.py

```diff
@@ -257,7 +257,27 @@ def tags(self) -> Dict[str, Any]:
 
     @property
     def nofailover(self) -> bool:
-        return self.tags.get('nofailover', False)
+        from_tags = self.tags.get('nofailover', None)
+        if from_tags is not None:
+            # Value of `nofailover` takes precedence over `failover_priority`
+            return bool(from_tags)
+        failover_priority = self.tags.get('failover_priority', 1)
+        return int(failover_priority) <= 0
+
+    @property
+    def failover_priority(self) -> int:
+        from_tags = self.tags.get('nofailover', None)
+        fp = self.tags.get('failover_priority', 1)
+        fp = int(fp) if fp is not None else 1
+        if from_tags is not None:
+            from_tags = bool(from_tags)
+            if from_tags is True:
+                # Value of `nofailover` takes precedence over `failover_priority`
+                return 0
+            elif fp <= 0:
+                # Value of `nofailover` takes precedence over `failover_priority`
+                return 1
+        return fp
```

Comment:
Something is wrong here...
```suggestion
        return 0 if from_tags else fp
```